ZukoAzula2016/src/org/team1540/zukoazula/DriveCode.java

```diff
@@ -18,21 +22,26 @@
     private static final ExtendedMotor rightFrontCAN = FRC.talonCAN(2), rightBackCAN = FRC.talonCAN(3);
     private static final ExtendedMotor leftFrontCAN = FRC.talonCAN(0), leftBackCAN = FRC.talonCAN(1);
 
-    public static void setup() throws ExtendedMotorFailureException {
-
-        final FloatOutput leftMotors = leftFrontCAN.simpleControl(FRC.MOTOR_FORWARD).combine(leftBackCAN.simpleControl(FRC.MOTOR_FORWARD));
-        final FloatOutput rightMotors = rightFrontCAN.simpleControl(FRC.MOTOR_REVERSE).combine(rightBackCAN.simpleControl(FRC.MOTOR_REVERSE));
+    private static final BehaviorArbitrator behaviors = new BehaviorArbitrator("Drive Code");
+    private static final ArbitratedFloat leftInput = behaviors.addFloat();
+    private static final ArbitratedFloat rightInput = behaviors.addFloat();
+    private static final Behavior teleop = behaviors.addBehavior("Teleop", FRC.inTeleopMode());
+    private static final Behavior pit = behaviors.addBehavior("Pit Mode", ZukoAzula.mainTuning.getBoolean("Pit Mode Enable", false));
 
-        final FloatOutput leftOut = leftMotors.addRamping(0.1f, FRC.constantPeriodic);
-        final FloatOutput rightOut = rightMotors.addRamping(0.1f, FRC.constantPeriodic);
+    public static void setup() throws ExtendedMotorFailureException {
+        leftInput.attach(teleop, driveLeftAxis.plus(driveLeftTrigger.minus(driveRightTrigger)));
+        rightInput.attach(teleop, driveRightAxis.plus(driveRightTrigger.minus(driveLeftTrigger)));
+        leftInput.attach(pit, FloatInput.zero);
+        rightInput.attach(pit, FloatInput.zero);
 
-        Drive.extendedTank(driveLeftAxis, driveRightAxis, driveLeftTrigger.minus(driveRightTrigger), leftOut, rightOut);
+        leftInput.send(leftFrontCAN.simpleControl(FRC.MOTOR_FORWARD).combine(leftBackCAN.simpleControl(FRC.MOTOR_FORWARD)).addRamping(0.1f, FRC.constantPeriodic));
+        rightInput.send(rightFrontCAN.simpleControl(FRC.MOTOR_REVERSE).combine(rightBackCAN.simpleControl(FRC.MOTOR_REVERSE)).addRamping(0.1f, FRC.constantPeriodic));
```

Comment:
Hmm. I think that @tytynguyen was right and I was wrong. Maybe move "a.combine(b)" into its own line.